dfastmi/gui/dialog_view.py

```diff
@@ -651,9 +657,14 @@ def _create_qthreshhold_input(
         self._qthr.setValidator(double_validator)
         self._qthr.editingFinished.connect(self._updated_qthreshold)
         self._qthr.setToolTip(gui_text("qthr_tooltip"))
-        qthrtxt = QLabel(gui_text("qthr"), self._win)
+
+        qthreshold_label_text = gui_text(
+            "qthr",
+            placeholder_dictionary={"stagnant": str(self._view_model.model.qthreshold)},
```

Comment:
you want to retrieve it from your viewmodel correct?
and your viewmodel will retrieve it from your model right?

Reply:
ai good catch, euh... need to think why i didn't get the view_model one

Reply:
I look into all .model properties if they are needed. plotting flags are also like this...